examples/spark/spark-gluster/spark-master-service.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
As I understand it, the "---" is used to signify the start of a document in yaml syntax.  I get this from the yaml documentation at http://www.yaml.org/spec/1.2/spec.html#id2760395